roles/openshift_registry/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+registry_volume: 'registry-volume'
```

Comment:
Is this used anywhere now that the PVs and PVCs are handled in openshift_persistent_volumes?